ssl/quic/quic_rcidm.c

```diff
@@ -250,6 +258,10 @@ static int rcid_cmp(const RCID *a, const RCID *b)
         return -1;
     if (a->seq_num > b->seq_num)
         return 1;
+    if ((uintptr_t)a < (uintptr_t)b)
+        return -1;
+    if ((uintptr_t)a > (uintptr_t)b)
+        return 1;
```

Comment:
This is a bit confusing to me.   Is this just computing the value in the a and b pointers?  Is there any meaning behind a being placed before or after b in memory?  I see this is a compare function for a priority queue, but is there something advantageous about making this comparison relevant rather than just calling the objects equal if the seq_nums are the same?